### DISCUSSION TITLE
Any way to disable the "Please enter deploy to confirm" question when deploying?

### DISCUSSION THREAD

**Discussion by billtorla-scopely** [NomicFoundation/hardhat#3239] [Troubleshooting]
I'm using hardhat in automation and would like to run it in a totally non-interactive fashion. Deploying asks the user to enter "deploy" in order to proceed. I got around this by piping printf to the hardhat command but I'd prefer a clean way of disabling the question entirely. Thanks in advance for your help!

`printf "deploy\n" | npx hardhat deploy --network $TARGET_NETWORK`

**Comment by fvictorio** [comment]
Hi @billtorla-scopely, I think that's a question for [hardhat-deploy](https://github.com/wighawag/hardhat-deploy).
